File: parcels/router/test/pages.js
export const pages = (() => {
  const START = "./pages".length;
  const pages = Object.fromEntries(
    Object.entries(import.meta.glob("./pages/**/*.js")).map(([k, load]) => {
      const path = `${k.slice(START, -3)}`;
      return [path, load];
    })
  );
  const home = pages[`/home`];
  if (home) {
    pages["/"] = home;
  }
  return pages;
})();

File: _history/server/server_code/20260424t1245/router/_pages.py
from ..tools import access, meta

# Default page
START = "start"

# NOTE Declared page:spec items
pages = dict(
    # Form routes
    iworker=dict(require=lambda: meta.PROD or access()),
    login=dict(),
    signup=dict(),
    start=dict(),
    test=dict(require=lambda: meta.DEV or (meta.PROD and access())),
    # View routes
    blog=dict(view="blog/blog.jinja"),
    foo=dict(view="foo/foo.jinja"),
)

# NOTE 'page' -> 'page' (useful for hyphenated form responses)
redirects = {
    "user-login": "login",
    "user-signup": "signup",
}


# NOTE 'path' -> 'path' (useful for compact page/view responses)
rewrites = {
    "/front": "/front/index.html",
   
}
